src/sonic-yang-models/yang-events/sonic-events-bgp.yang

```diff
@@ -0,0 +1,97 @@
+module sonic-events-bgp {
+    namespace "http://github.com/Azure/sonic-events-bgp";
+
+    yang-version 1.1;
+
+    import sonic-events-common {
```

Comment:
If common will also have a revision, if yes kindly add revision here.

Reply:
YANG review is moved to different [PR #11404 ](https://github.com/Azure/sonic-buildimage/pull/11404)